sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -62,6 +65,14 @@ class SchemaRegistryClient(object):
 
     def __init__(self, fully_qualified_namespace, credential, **kwargs):
         # type: (str, TokenCredential, Any) -> None
+        api_version = kwargs.get("api_version")
+        if api_version and api_version not in SUPPORTED_API_VERSIONS:
+            versions = "\n".join(SUPPORTED_API_VERSIONS)
+            raise ValueError(
+                "Unsupported API version '{}'. Please select from:\n{}".format(
+                    api_version, versions
+                )
+            )
```

Comment:
we don't handle api version at our application level, we pass it to the underlying azure-core

you could also refer to the search for api_version:

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/search/azure-search-documents/azure/search/documents/_search_client.py#L65-L105